docs/upgrade.html

```diff
@@ -34,9 +34,11 @@ <h5><a id="upgrade_420_notable" href="#upgrade_420_notable">Notable changes in 4
     <li>The <code>KafkaPrincipalBuilder</code> now extends <code>KafkaPrincipalSerde</code>. Force developer to implement <code>KafkaPrincipalSerde</code> interface for custom <code>KafkaPrincipalBuilder</code>.
         For further details, please refer to <a href="https://cwiki.apache.org/confluence/x/1gq9F">KIP-1157</a>.
     </li>
+    <li>
+        The support for MX4J library, enabled through <code>kafka_mx4jenable</code> system property, was deprecated and will be removed in the next  major release.
```

Comment:
We have a redundant space here.
```suggestion
        The support for MX4J library, enabled through <code>kafka_mx4jenable</code> system property, was deprecated and will be removed in the next major release.
```

Reply:
Thanks for spotting this, fixed.